docs/pages/blog/first-look-at-joy.md

```diff
@@ -0,0 +1,133 @@
+
+Material UI and Joy UI will still share many conventions and features.
+This will not only decrease the learning curve for adopting Joy, but also because we think of them both representing the ideal developer experience for developing design systems.
+So if at any time Joy UI has a feature that Material UI doesn't, it will catch-up at some point, and vice versa.
+
+## What are the new things we're introducing in Joy UI?
+
+Aside from a fresher look and feel for your upcoming project, here are a few features we're planning for the alpha version of Joy UI:
+
+### Global variants
+
+Instead of defining variants at the component level, Joy UI defines variants at the global level.
+This sounds bold at first, but the more we experiment the more it starts to make sense.
+[Global variants](/joy-ui/core-features/global-variant/) mostly affect three CSS properties: color, background, and border.
+
+What this means is that you'll be able to switch between the same set of variants in any component, more easily maintaining consistency across your app.
+The four variants available in the components are `plain`, `outlined`, `soft`, and `solid`.
+
+<img src="/static/blog/first-look-at-joy/global-variants.png" style="width: 796px; margin-top: 16px; margin-bottom: 16px;" alt="Screenshot of button component using all available variants." />
+
+```jsx
+<Button variant="solid">
+<Button variant="soft">
+<Button variant="outlined">
+<Button variant="plain">
+```
+
+### Component integration
+
+We put considerable effort into constructing the CSS variables for each component such that they can be seamlessly composed with the least customization.
+One good example is the input component, where the border radius of the input's children automatically adapts to that of the input.
+These small details mean the components adapt to different scenarios, which saves considerable time by avoiding manual adjustments.
+
+<img src="/static/blog/first-look-at-joy/component-integration.png" style="width: 796px; margin-top: 16px; margin-bottom: 8px;" alt="Screenshot of two text inputs, one being native from Joy UI and another with border-radius customized" />
+
+<p class="blog-description">When customizing the input's border radius, the icon button inside of it adapts automatically.</p>
+
+```jsx
+<Input
+  placeholder="password"
+  endDecorator={<IconButton size="sm"><Visibility /></IconButton>}
+/>
+
+<Input
+  size="lg"
+  placeholder="password"
+  endDecorator={<IconButton><Visibility /></IconButton>}
+  sx={{ '--Input-radius': '24px' }}
+/>
+```
+
+Joy UI doesn't sacrifice the customization experience in order to have these automatic adjustments.
+You're still able to override the style completely via the usual CSS overrides, or even adjust the predefined CSS variables with the `sx` prop.
+
+### Perfect dark mode, for server-side rendering
+
+Joy UI provides a simple and effective way to prevent UI flicker when users refresh or re-enter the page when dark mode is enabled.
```

Comment:
Iterating to cut "simple" and avoid the double "when"s.

```suggestion
Joy UI provides an effective way to prevent UI flicker when users refresh or re-enter a page with dark mode enabled.
```